src/main/java/org/fcrepo/transform/transformations/LDPathTransform.java

```diff
@@ -157,15 +157,4 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Objects.hashCode(query);
```

Comment:
it's also worth noting that, at this point, it is substantially unnecessary for the LDPath code to even be running in the same container as Fedora.